client/utils/data.js

```diff
@@ -1,7 +1,15 @@
 var axios = require('axios');
 const current = process.env.NODE !== '/app/.heroku/node/bin/node' ? 'http://localhost:1337' : 'https://postgress-express-backend.herokuapp.com'
+let formUUID;
```

Comment:
@cofuente I see what you mean. Sometimes, the one-liner/short form is not readable for me. This will most likely be handled another way in the future.

Reply:
@drakeltheryuujin cool beans!